wgpu-hal/src/dx12/mod.rs

```diff
@@ -614,18 +614,22 @@ impl crate::Surface<Api> for Surface {
         // We always set ALLOW_TEARING on the swapchain no matter
         // what kind of swapchain we want because ResizeBuffers
         // cannot change if ALLOW_TEARING is applied to the swapchain.
```

Comment:
Pre-existing, but: 
```suggestion
        // cannot change the swapchain's ALLOW_TEARING flag.
```

The original text can easily be read as:

> if `ALLOW_TEARING` is appled to the swapchain, then `ResizeBuffers` cannot change

which is meaningless and confusing.